activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/rest/validation/GenericModelPayloadValidator.java

```diff
@@ -30,11 +30,17 @@
     public static final String MODEL_INVALID_NAME_EMPTY_MESSAGE =
             "The model name cannot be empty";
 
-    private final boolean checkRequiredFields;
+    public static final int MODEL_NAME_MAX_LENGTH = 26;
 
-    public GenericModelPayloadValidator(boolean checkRequiredFields) {
-        this.checkRequiredFields = checkRequiredFields;
-    }
+    public static final String MODEL_INVALID_NAME_LENGTH_MESSAGE =
+            "The model name length cannot be greater than " + MODEL_NAME_MAX_LENGTH;
+
+    public static final String DNS_LABEL_REGEX = "^[a-z]([-a-z0-9]{0,24}[a-z0-9])?$";
```

Comment:
If you keep "{0, 24}" condition then you will receive two validation errors for long names: both "The model name length cannot be greater than 26" and "The model name should follow DNS-1123 conventions: [...]". Either remove "{0, 24}" condition or the dedicated validation on length. I would go with removing the "{0, 24}" condition.